test/integration/delete-resource/index.js

```diff
@@ -0,0 +1,31 @@
+import {expect} from "chai";
+import AgentPromise from "../../app/agent";
+import mongoose from "mongoose";
+
+describe("Deleting a resource", () => {
+
+  let id;
+  before(done => {
+    AgentPromise.then(Agent => {
+      mongoose.models.Organization.create({ name: "Delete me" }, (err, model) => {
```

Comment:
Save story as my prior comment...this should be a POST and then a DELETE and then a GET, rather than using mongoose directly.

Thanks for adding these tests, btw! We've needed delete ones...